frontend/catalyst/compiler.py

```diff
@@ -264,6 +264,16 @@ def get_default_flags(options):
         mlir_lib_path = get_lib_path("llvm", "MLIR_LIB_DIR")
         rt_lib_path = get_lib_path("runtime", "RUNTIME_LIB_DIR")
 
+        # Adds RUNTIME_LIB_DIR to the Python system path to allow the catalyst_callback_registry
+        # to be importable.
+        sys.path.append(get_lib_path("runtime", "RUNTIME_LIB_DIR"))
+        import catalyst_callback_registry as registry  # pylint: disable=import-outside-toplevel
```

Comment:
I assume there's good reason to avoid importing this at top level?

Reply:
We currently lack a "top level initialization" of Catalyst, but yes I agree that if we had one, we could append the runtime directory to the sys path and then be able to import it. Happy for this to be a follow up item.

Reply:
I thought we did some "top level initialization" in `catalyst/__init__.py`?

Reply:
I guess you are right! I very rarely open this file :sweat_smile:

Reply:
But then `get_lib_path` needs to be moved to init. I think we can formalize this a bit more and move `mlir_lib_path` and `rt_lib_path` to the `__init__.py` file as well. Could this be done as another ticket to gather all initialization procedure in a single function?